src/test/java/study/step4/RacingGameProgressTest.java

```diff
@@ -0,0 +1,24 @@
+package study.step4;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class RacingGameProgressTest {
+
+    @ParameterizedTest
+    @DisplayName("게임 구현에 대한(결과값) 테스트")
+    @ValueSource(ints = {10, 25, 30})
+    void gameStartTest(int tryNumber){
+        String carNames ="차1,차2,차3";
+        RacingGame racingGame = new RacingGame(carNames,tryNumber);
+        RacingGameProgress racingGameProgress = new RacingGameProgress(racingGame);
+        List<Round> rounds = racingGameProgress.gameStart(new RandomCarMoveCondition());
+
+        assertThat(rounds.size()).isEqualTo(tryNumber);
```

Comment:
```suggestion
        assertThat(rounds).hasSize(tryNumber);
```
한가지 팁인데, assertThat은 Collection 관련 많은 기능을 제공해줍니다 ㅎ